lib/cc/cli/command.rb

```diff
@@ -7,6 +7,50 @@ module CC
   module CLI
     class Command
       CODECLIMATE_YAML = ".codeclimate.yml".freeze
+      NAMESPACE = name.split("::")[0..-2].join("::").freeze
+
+      def self.abstract!
+        @abstract = true
+      end
+
+      def self.abstract?
+        @abstract == true
```

Comment:
Lately I'm more inclined to use proper types. Query methods like this are supposed to return a boolean value. In Ruby that is either `true` or `false`. This variable is specifically set to `true` and there's no API to set it to anything else so all other values (including default `nil`) are `false`.

Getting `nil` from query method feels more like 🤷‍♀️ for an answer to a yes/no question.

I personally might've written `!!@abstract` but Rubocop doesn't like that.